### DISCUSSION TITLE
NECESITO AYUDA (I NEED HELP)

### DISCUSSION THREAD

**Discussion by ricitos2001** [community/community#114610] [Other Feature Feedback, Questions, & Ideas]
ES: Estoy buscando un probador que pueda identificar si hay algún error en los programas que he creado
¿Algún voluntario?
EN: I'm looking for a tester who can identify if there are any bugs in the programs I've created.
Any volunteers?

**Comment by mairinkdev** [comment]
👋

**Reply by ricitos2001** [reply]
👋

**Reply by ricitos2001** [reply]
can you help me?

**Comment by mairinkdev** [comment]
sure, what you need?

**Reply by ricitos2001** [reply]
i need a tester for help me to find if my programs have errors or else
i uploaded the programs in the repositories
can you help me?

**Comment by mairinkdev** [comment]
I didn't understand exactly, would it be open source code and do you need
help with the code? because just testing you can do this with some other
tools
